src/Card/Card.js

```diff
@@ -13,6 +13,10 @@ class Card extends Component {
      */
     children: PropTypes.node,
     /**
+     * Override the inline-styles of the children element.
+     */
+    childrenStyle: PropTypes.object,
```

Comment:
I'm wondering if `containerStyle` wouldn't be better. I would be using `childrenStyle` if we were cloning the `children`.